From 2368d81b9931d07408f8f4fd5f2ffda29d802bb4 Mon Sep 17 00:00:00 2001
From: Andreas Pehrson <pehrson@comoyo.com>
Date: Fri, 25 Apr 2014 09:22:12 +0200
Subject: [PATCH] Disable OOP in Browser

---
 apps/browser/js/browser.js | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/apps/browser/js/browser.js b/apps/browser/js/browser.js
index 9882395..c4b88f5 100644
--- a/apps/browser/js/browser.js
+++ b/apps/browser/js/browser.js
@@ -1234,7 +1234,7 @@ var Browser = {
   createTab: function browser_createTab(url, iframe, tab) {
     if (!iframe) {
       iframe = document.createElement('iframe');
-      iframe.setAttribute('mozbrowser', true);
+      iframe.setAttribute('mozbrowser', false);
       iframe.setAttribute('mozallowfullscreen', true);
       iframe.classList.add('browser-tab');
 
@@ -1247,7 +1247,7 @@ var Browser = {
 
     iframe.setAttribute('mozasyncpanzoom', 'true');
     // FIXME: content shouldn't control this directly
-    iframe.setAttribute('remote', 'true');
+    iframe.setAttribute('remote', 'false');
 
     if (tab) {
       tab.dom = iframe;
-- 
1.9.0

