From 15e02f376f4a2ae1fe6d4df12dd1631f9ae21b43 Mon Sep 17 00:00:00 2001
From: Andreas Pehrson <pehrson@comoyo.com>
Date: Tue, 20 May 2014 09:31:49 +0200
Subject: [PATCH] Use EXTRA_*_FLAGS when building gonk

---
 core/combo/select.mk | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/core/combo/select.mk b/core/combo/select.mk
index e485d00..5c51b67 100644
--- a/core/combo/select.mk
+++ b/core/combo/select.mk
@@ -46,9 +46,9 @@ $(combo_target)HAVE_STRLCPY := 0
 $(combo_target)HAVE_STRLCAT := 0
 $(combo_target)HAVE_KERNEL_MODULES := 0
 
-$(combo_target)GLOBAL_CFLAGS := -fno-exceptions -Wno-multichar
-$(combo_target)RELEASE_CFLAGS := -O2 -g -fno-strict-aliasing
-$(combo_target)GLOBAL_LDFLAGS :=
+$(combo_target)GLOBAL_CFLAGS := -fno-exceptions -Wno-multichar $($(combo_target)EXTRA_GLOBAL_CFLAGS)
+$(combo_target)RELEASE_CFLAGS := -O2 -g -fno-strict-aliasing $($(combo_target)EXTRA_RELEASE_CFLAGS)
+$(combo_target)GLOBAL_LDFLAGS := $($(combo_target)EXTRA_GLOBAL_LDFLAGS)
 $(combo_target)GLOBAL_ARFLAGS := crsP
 
 $(combo_target)EXECUTABLE_SUFFIX :=
-- 
1.9.0

